tachyon/crypto/commitments/kzg/kzg_unittest.cc

```diff
@@ -55,6 +55,44 @@ TEST_F(KZGTest, CommitLagrange) {
   EXPECT_EQ(commit, commit_lagrange);
 }
 
+TEST_F(KZGTest, BatchCommitLagrange) {
+  using Domain = math::UnivariateEvaluationDomain<math::bn254::Fr, kMaxDegree>;
+  using Poly = math::UnivariateDensePolynomial<math::bn254::Fr, kMaxDegree>;
+  using Evals = math::UnivariateEvaluations<math::bn254::Fr, kMaxDegree>;
+
+  PCS pcs;
+  ASSERT_TRUE(pcs.UnsafeSetup(N));
+
+  size_t num_polys = 10;
+  std::vector<Poly> polys =
+      base::CreateVector(num_polys, []() { return Poly::Random(N - 1); });
+
+  BatchCommitmentState state;
+  state.batch_mode = true;
+  state.batch_count = num_polys;
+  for (size_t i = 0; i < num_polys; ++i) {
+    ASSERT_TRUE(
+        pcs.Commit(polys[i].coefficients().coefficients(), state, nullptr));
+  }
+  std::vector<math::bn254::G1AffinePoint> batch_commitments =
+      pcs.batch_commitments(state);
+
```

Comment:
Please add a check

```suggestion
  EXPECT_EQ(state.batch_mode, false);
  EXPECT_EQ(state.batch_count, size_t{0});
```

Reply:
I've also added a check for batch index as well!